src/main/java/org/broadinstitute/hellbender/engine/filters/OverclippedReadFilter.java

```diff
@@ -26,7 +26,7 @@
 
     @Argument(fullName = "filterTooShort",
             shortName = "filterTooShort",
-            doc = "Value for which reads with less than this number of aligned bases is considered too short",
+            doc = "Value for which reads with less than this number of aligned bases is considered too short.",
```

Comment:
Done. Maybe the name could be also simplified to something like `--maximunNoClippedLength`, but this will be a non-backwards compatible change...